### PR TITLE
Feature/sc 463/cleanup internal group view

### PR DESCRIPTION
**Pull request by Plebbimon**

### CODE REVIEW

File: organization/schema.py [hunk 1]
```diff
@@ -331,6 +344,70 @@ def mutate(self, info, membership_id, *args, **kwargs):
...
+    image = graphene.String()
+
+    def resolve_image(self: InternalGroupUserHighlight, info, **kwargs):
+        if self.image:
+            return self.image.url
+        else:
+            return None
+
+    @classmethod
+    def get_node(cls, info, id):
```
Comment (alexaor) [inline organization/schema.py]: Add a gql login decorator to this

```suggestion
    @classmethod
    @gql_login_required()
    def get_node(cls, info, id):
```

File: organization/schema.py [hunk 2]
```diff
@@ -331,6 +344,70 @@ def mutate(self, info, membership_id, *args, **kwargs):
...
+class DeleteInternalGroupUserHighlight(DjangoDeleteMutation):
+    class Meta:
+        model = InternalGroupUserHighlight
+        permissions = ("organization.delete_internalgroupuserhighlight",)
+
+
+class BatchPatchInternalGroupUserHighlightMutation(DjangoBatchPatchMutation):
+    class Meta:
+        model = InternalGroupUserHighlight
+        permissions = ("organization.change_internalgroupuserhighlight",)
```
Comment (alexaor) [inline organization/schema.py]: Will we be using this?

File: organization/schema.py [hunk 3]
```diff
@@ -331,6 +344,70 @@ def mutate(self, info, membership_id, *args, **kwargs):
...
+    internal_group_user_highlight = Node.Field(InternalGroupUserHighlightNode)
+    internal_group_user_highlights_by_internal_group = graphene.List(
+        InternalGroupUserHighlightNode, internal_group_id=graphene.ID()
+    )
+
+    @gql_has_permissions("organization.view_internalgroupuserhighlight")
+    def resolve_all_internal_group_user_highlights(self, info, *args, **kwargs):
+        return InternalGroupUserHighlight.objects.all()
+
+    @gql_has_permissions("organization.view_internalgroupuserhighlight")
```
Comment (alexaor) [inline organization/schema.py]: Change to login required

**Review by alexaor — CHANGES_REQUESTED** [review]

File: organization/schema.py (hunk 1 above)
Comment (Plebbimon) [reply organization/schema.py]: done

File: organization/schema.py (hunk 2 above)
Comment (Plebbimon) [reply organization/schema.py]: no.

**Review by alexaor — APPROVED** [review]

**Pull request merged**